src/components/ImageCarousel.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'; 
+
+import projectsData from '../resources/projectsData'; 
+
+function ImageCarousel() {
+    return (
+        <div className="composition">
```

Comment:
Previously I had this all hard coded into three separate <a><img/></a> JSX segments. This looks a lot cleaner, and easier to update!